components/compliance-service/ingest/projectupdater/manager.go

```diff
@@ -144,47 +154,76 @@ func (manager *Manager) waitingForJobToComplete() {
 	)
 
 	// initial status
-	manager.sendStatusEvent(ingestic.JobStatus{})
+	manager.updateStatus(ingestic.JobStatus{})
 
-	jobStatus, err := manager.client.JobStatus(context.Background(), manager.esJobID)
+	jobStatuses, err := manager.collectJobStatus()
 	if err != nil {
 		logrus.Errorf("Failed to check the running job: %v", err)
 		numberOfConsecutiveFails++
 	} else {
-		manager.sendStatusEvent(jobStatus)
-		isJobComplete = jobStatus.Completed
+		mergedJobStatus := MergeJobStatus(jobStatuses)
+		isJobComplete = mergedJobStatus.Completed
+		manager.updateStatus(mergedJobStatus)
 	}
 
 	for !isJobComplete {
```

Comment:
nitpick: I think you could reduce a little bit of complexity here by not using `isJobComplete`, moving `mergedJobStatus` to the outer scope, and changing this condition to `!mergedJobStatus.Completed`.

Reply:
That is a good point! Thanks

Reply:
I update it with this commit. https://github.com/chef/automate/pull/86/commits/6d58c39cc7d950f0538c62a141dda27096a3688b